### PR TITLE
Add etherscan link for pending txs in transaction modal

### PR DESCRIPTION
**Pull request by NakedFool**
## What does this PR do and why?

Fixes - https://github.com/jbx-protocol/juice-interface/issues/2335

I'm using `TxHistoryContext` to retrieve transactions, and from there i'm just finding the first one that matches the pending status.

## Screenshots or screen recordings

I've created 3 iterations, i'm a fan of the first one but let me know which one we should decide or if we want some different UI iteration.

| First iteration| Second Iteration | Third Iteration
| ----------- | ----------- | ----------- | 
| first iteration     | second iteration| third iteration     |

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [x] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/TransactionModal.tsx [hunk 1]
```diff
@@ -42,8 +48,11 @@ const PendingTransactionModalBody = () => {
         </h2>
         <p>
           <Trans>
-            Your transaction has been submitted and is awaiting confirmation.
-          </Trans>
+            Your transaction has been submitted and is awaiting confirmation{' '}
+          </Trans>{' '}
+          {pendingTx ? (
+            <EtherscanLink value={pendingTx.tx?.hash} type="tx" />
```
Comment (tomquirk) [inline src/components/TransactionModal.tsx:54]: **suggestion** wdyt about a text link "Go to Etherscan" or "View in Etherscan" instead?

Comment (NakedFool) [reply src/components/TransactionModal.tsx]: yup, i've chosen "View in Etherscan"

updated in the latest commit ser 🍻

Comment (NakedFool) [reply src/components/TransactionModal.tsx]: image

File: src/components/TransactionModal.tsx [hunk 2]
```diff
@@ -42,9 +49,24 @@ const PendingTransactionModalBody = () => {
         </h2>
         <p>
           <Trans>
-            Your transaction has been submitted and is awaiting confirmation.
+            Your transaction has been submitted and is awaiting confirmation{' '}
           </Trans>
         </p>
+        <p>
+          {pendingTx ? (
+            <ExternalLink
```
Comment (tomquirk) [inline src/components/TransactionModal.tsx:57]: **suggestion** Could we use EtherscanLink here perhaps?

`src/components/EtherscanLink.tsx`

**Comment by tomquirk** [comment]
Thanks @NakedFool looks great! One last comment

File: src/components/TransactionModal.tsx (hunk 2 above)
Comment (NakedFool) [reply src/components/TransactionModal.tsx]: Yeah, we can use `<EtherscanLink/>` but that component also returns an icon so it looks like this.
I've added children inside that component if `EtherscanLink` is a type of `tx`
image

It looks cool to me, wdyt?

Comment (NakedFool) [reply src/components/TransactionModal.tsx]: We can put logic to remove the icon if there are children, but this looks good to me the way it is! :)

**Review by tomquirk — APPROVED** [review]
LGTM @NakedFool !!!

Added one commit, but merging. 💪  https://github.com/jbx-protocol/juice-interface/pull/2376/commits/5e5d299a5270f3375543842eeb71a6a510289596

**Pull request merged**